docs/scripts/api/buildComponentsDocumentation.ts

```diff
@@ -250,20 +250,33 @@ const buildComponentDocumentation = async (options: {
     reactApi.styles.globalClasses[key] = globalClass;
   });
 
-  const allProjectsName = Array.from(projects.keys());
-  const projectsWithThisComponent = allProjectsName.filter((projectName) => {
-    const currenetProject = projects.get(projectName) as Project;
-    const symbol = currenetProject.exports[reactApi.name];
+  reactApi.packages = Array.from(projects.keys())
```

Comment:
This modification allow to have the correct import with `Unstable_` on the api page

Reply:
The whole block

We need to get the symbol (otherwise we have no import at all on the api page) and we need to pass the name of the export (otherwise the import would not have the `Unstable_` prefix)